src/backend/distributed/commands/database.c

```diff
@@ -100,6 +101,15 @@ PostprocessAlterDatabaseOwnerStmt(Node *node, const char *queryString)
 		return NIL;
 	}
 
+	bool createInProgress = true;
+	DeferredErrorMessage *errMsg = DeferErrorIfHasUnsupportedDependency(&typeAddress,
+																		createInProgress);
```

Comment:
why does ALTER DATABASE .. OWNER TO need this?

Reply:
it doesn't, dropped those lines now